src/controller/binding.hxx

```diff
@@ -53,6 +53,7 @@ public:
 	int track;
 	int scene;
 	int send;
+	int beat;
```

Comment:
At some point it may be useful to introduce a union here, but that's a bit too involved for the current PR.

Reply:
Agree - no need to refactor to union here. I prefer using a little more memory per message/binding, and keep things simpler.